tools/templates/DIGESTS.md

```diff
@@ -0,0 +1,23 @@
+## Subresource Integrity
+
+If you are loading Highlight.js via CDN you may wish to use [Subresource Integrity](https://developer.mozilla.org/en-US/docs/Web/Security/Subresource_Integrity) to guarantee that you are using a legimitate build of the library.
+
+To do this you simply need to add a `digest` attribute for each JavaScript file you download via CDN. These digests are used by the browser to confirm the files downloade have not been modified.
+
+```html
+<script
+  src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/<!-- $VERSION -->/highlight.min.js"
+  digest="<!-- $MIN_JS_DIGEST -->"></script>
```

Comment:
```suggestion
  integrity="<!-- $MIN_JS_DIGEST -->"></script>
```

Reply:
LOL. Oops. Not sure how I made that mistake.